package.json

```diff
@@ -53,13 +53,14 @@
     "ansi-colors": "3.2.3",
     "ast-replace": "1.1.3",
     "atob": "2.1.2",
-    "ava": "1.0.1",
     "autoprefixer": "9.4.3",
+    "ava": "1.0.1",
     "babel-eslint": "10.0.1",
     "babel-plugin-filter-imports": "2.0.4",
     "babel-plugin-istanbul": "5.1.0",
     "babel-plugin-transform-commonjs-es2015-modules": "4.0.1",
     "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-regenerator-runtime": "6.5.0",
```

Comment:
Intentional addition of `babel-regenerator-runtime`?

Reply:
Must've been a bad merge. Removed, thanks.